python/hail/matrixtable.py

```diff
@@ -777,7 +648,12 @@ def key_rows_by(self, *keys, **named_keys) -> 'MatrixTable':
         keys = get_select_exprs("MatrixTable.key_rows_by",
                              keys, named_keys, self._row_indices,
                              protect_keys=False)
-        return self._key_rows_by("MatrixTable.key_rows_by", keys, {})
+
```

Comment:
I see. The logic is not as complicated now, so it doesn't need a separate method.